docs/implementation/vertical-slice.md

```
@@ -0,0 +1,254 @@
+
+```bash
+# 터미널 1: daemon 시작
+bun run --cwd apps/daemon dev
+
+# 터미널 2: CLI chat 시작
+bun run --cwd apps/cli start chat
+```
+
+예시 대화:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)